COMP 120 Tinkering Graphics-JH182233/Program/Option.py

```diff
@@ -0,0 +1,205 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
```

Comment:
Using 'global' is only necessary when defining a variable inside a function or class. Even then, it is often better practice not to give variables inside functions global scope. It makes the code more difficult to maintain as any function can change that variable.